### PR TITLE
WIP Adds noAck option

### PR DESCRIPTION
**Pull request by jacktuck**
WIP having a crack at #18 

Went for not having a separate endpoint as would be preferable i would imagine?

Not sure how close this is to landing.

### CODE REVIEW

**Comment by jacktuck** [comment]
TODO Send `done` as a `noop` when `noAck` is `true`

File: index.js [hunk 1]
```diff
@@ -74,13 +74,24 @@ Remit.prototype.res = function res (event, callbacks, context, options) {
         self.__assert_exchange(() => {
             const chosen_queue = options.queueName || event
 
+            const queueOptions = {
```
Comment (jpwilliams) [inline index.js]: With these options, can we change the check to be `options.hasOwnProperty('xyz')` and then, if `true`, coerce the value to a boolean using `!!`?

**Comment by jpwilliams** [comment]
I _love_ the ability added to provide the options and we'll keep that, but I do also really want a new endpoint type, `.ares()`.

This is a use case that could crop up quite a bit and I'd rather people just use that endpoint type rather than having to look up RabbitMQ and `noAck` usage.

I'll add this in - shouldn't be a problem!

**Comment by jpwilliams** [comment]
Right. Needs lots of testing.

**Comment by jacktuck** [comment]
LGTM. Yeah it does.

**Comment by jpwilliams** [comment]
If we're doing `.ares()`, we might as well do `.alisten()` too.

**Comment by jacktuck** [comment]
Can we land this? Currently our service-pusher is running on my fork.

**Pull request merged**